nixos/modules/services/networking/birdsitelive.nix

```
@@ -0,0 +1,442 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.birdsitelive;
+  settingsType = (pkgs.formats.json { }).type;
+
+  isSecret = v: with builtins; isAttrs v && v ? _secret && isString v._secret;
+  secret = with lib; mkOptionType {
+    name = "secret";
+    description = "secret value";
+    descriptionClass = "noun";
+    check = isSecret;
+    getSubOptions = prefix: prefix ++ [ "_secret" ];
+    nestedTypes = {
+      _secret = types.str;
+    };
+  };
+
+  toJsonSec =
+    let toJsonSec' = {}@args: v: with builtins;
```

```suggestion
    let toJsonSec' = {}: v: with builtins;
```
args is always empty

---

Corrected.

---

🤔 I don't know, should be fine then